src/env.testing

```diff
@@ -0,0 +1,68 @@
+APP_NAME="CoolStuff Portal"
+APP_ADMIN_NAME="CoolStuff Portal Admin"
+APP_ENV=local
+APP_KEY=base64:Ig1MAu4Go0YyDh+0tDByQKyi9V+NJILnaNMZDkuHArg=
+APP_DEBUG=true
+APP_URL=http://0.0.0.0:9001/
+
+LOG_CHANNEL=daily
+LOG_DEPRECATIONS_CHANNEL=null
+LOG_LEVEL=debug
+
+DB_CONNECTION=mysql_testing
+DB_HOST=mysql_testing
+DB_DATABASE=tribes_db
```

Comment:
@scruffycoderofficial The database name does not correspond with the setting within `phpnit.xml` file. These values must correspond together.